src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProvider.java

```diff
@@ -80,15 +84,43 @@
     @Override
     public Set<Instance> createInstance(Infrastructure infrastructure, Instance instance) {
 
+        String instanceImageId;
+        ResourcePool destinationPool = null;
+        HostSystem destinationHost = null;
+        Datastore destinationDatastore = null;
+        Folder destinationFolder = null, finalDestinationFolder;
+
         try {
+            Folder rootFolder = vmWareServiceInstanceCache.getServiceInstance(infrastructure).getRootFolder();
 
-            String instanceFolder = instance.getImage().split("/")[0];
-            String instanceImageId = instance.getImage().split("/")[1];
+            if (instance.getImage().contains("/")) {
+                instanceImageId = instance.getImage().split("/")[0];
+                String hostname = instance.getImage().split("/")[1];
+                if (hostname.equals("*")) {
```

Comment:
I created a dedicated method to check size > 1, see 29a118819496c5464cc0fd26ad78da39f73a0fa2